### ISSUE TITLE
BufferFactory not implemented ?

### ISSUE THREAD

**Issue by yves-surrel** [basler/pypylon#219]
In order to use instant_camera.SetBufferFactory, one needs to subclass a pylon.BufferFactory that does not seem to be accessible or implemented anywhere...

**Comment by thiesmoeller** [comment]
Yes. Bufferfactory is not properly wrapped at the moment. As we currently focussed on hiding all memory handling from the user.

What is your use case? 
So we could check what kind of mapping might solve your application needs.

**Comment by yves-surrel** [comment]
I want to use an OpenCL mapped memory or an OpenGL buffer. Would this be possible ?

Yves

**Comment by Filippo221** [comment]
Hello, has this been fixed? 

Thanks!
